spec/lib/api/2.0/notification-spec.js

```diff
@@ -264,5 +264,26 @@ describe('Http.Api.Notification', function () {
                 .expect(400);
             });
         });
+
+        describe('POST /notification/alerts', function () {
+            var alert = {
+                '{"Severity":"Critical"}': "d"};
+            before(function(){
+                sinon.stub(notificationApiService, 'redfishAlertProcessing').resolves();
+            });
+
+            it('should post alert notification successfully', function () {
+                return helper.request()
+                    .post('/api/2.0/notification/alerts')
+                    .set('Content-Type', 'application/x-www-form-urlencoded')
```

Comment:
Since you specify the API will consume `application/json` as well. so please create a test case to make sure the `application/json` works as well.